protocol/contracts/amo/Ramos.sol

```diff
@@ -191,207 +264,304 @@ contract RAMOS is Ownable, Pausable {
      * @param to Recipient address
      * @param amount Amount to recover
      */
-    function recoverToken(address token, address to, uint256 amount) external onlyOwner {
+    function recoverToken(address token, address to, uint256 amount) external onlyElevatedAccess {
```

Comment:
To confirm, are you suggesting that we always send the funds to the 'executor' or 'rescuer' role, rather than as a `to` param?

In the case that someone accidentally transferred a token to this contract and want it back, having the `to` allows us to send it straight back to the user, rather than multiple tx's.

`recoverToken` would be an example for a higher threshold of signers (eg 3 or 4)

Reply:
no, I wasn't suggesting we send to the `executor/rescuer`, i meant a predefined mutable state variable (destination ) that could be set be elevatedAccess.
Considering the latter case of user locked funds, having it unchanged skips an extra transfer. So you can leave it as is